### ISSUE TITLE
Rules Scheduler: task list is missing

### ISSUE THREAD

**Issue by bugfolder** [backdrop-contrib/rules#215]
On the Schedule tab of the Rules Scheduler sub-module, at admin/config/workflow/rules/schedule, even after you have scheduled a task, there are no tasks listed:

![image](https://github.com/backdrop-contrib/rules/assets/29214084/4d94a7f6-08cb-4dfc-8295-6fcf8ce2c87b)

However, they're there in the D7 version of Rules:

![image](https://github.com/backdrop-contrib/rules/assets/29214084/943bf245-60a2-413c-9583-5ed980229311)

We should restore this table to the Backdrop Rules Schedule page.

**Comment by bugfolder** [comment]
Well, that was easy. The Rules Schedule view was missing. 

![image](https://github.com/backdrop-contrib/rules/assets/29214084/b17c30fa-2f13-4929-98c4-fc7a01fe56b2)

It's typically brought over with a D7 upgrade, but is missing from new installs. Looks like we need a config file in `rules_scheduler` module and an update hook for existing installations.

**Comment by argiepiano** [comment]
Thanks @bugfolder - please see comments in the PR.

**Comment by argiepiano** [comment]
Thanks for the changes. I still am unsure if reading a file in an update hook is a good idea. I guess we'll find out. The PR worked fine in my very basic testing. LGMT.

**Comment by bugfolder** [comment]
>  I still am unsure if reading a file in an update hook is a good idea.

I think the main difference would be that if we someday update the default View in config, when we're reading it in from the file, the update hook will get the new version, whereas if we duplicated its content in the update hook, then even after updating the default view, updates would get the old one (unless we updated the update hook, or added a new update hook).

Under the DRY principle, seems like reading it in from the file in the update hook ismore robust than putting the same long, complex block of code in two places.

Since both `backdrop_get_path()` and `backdrop_json_decode()` are in `common.inc`, we can count on them being available to update hooks.
